src/helpers.js

```diff
@@ -250,23 +251,10 @@ export function getFormatedDateTime (timestamp) {
   }
 }
 
-export function formatDateFromNow (d, msg) {
-  const now = Date.now()
-  const date = new Date(d)
-
-  const suffixes = [ msg.secondsShort(), msg.minutesShort(), msg.hoursShort(), msg.daysShort(), msg.monthsShort(), msg.yearsShort() ]
-  const divitions = [ 1000, 60, 60, 24, 30, 12 ]
-  let elapsed = (now - date.getTime())
-  let suffix = ''
-
-  let currentIndex = 0
-  do {
-    suffix = suffixes[currentIndex]
-    elapsed = Math.floor(elapsed / divitions[currentIndex])
-    currentIndex++
-  } while (divitions[currentIndex] <= elapsed && currentIndex < suffixes.length)
-
-  return msg.timeAgo({ time: `${elapsed}${suffix}` })
+export function formatDateFromNow (date, msg, now = new Date()) {
```

Comment:
A few nit-picks:

  - I suggest moving this function to `src/utils/format.js` to be in the same place as `formatUptimeDuration`.

  - Maybe a slightly different name?  `formatHowLongAgo`

  - `msg` isn't needed since `humanize(true)` handles that for us

  - `s/date/pastDate/` maybe?